packages/core/test/tools/create-http-patch.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
```

Comment:
This file is actually just `packages/core/test/tools/http-patch.js` renamed to be equivalent to the name of the file it's testing. I also removed a top-level `describe` and added two new tests to cover the changes in this PR.